### ISSUE TITLE
Upgrade to grpc 1.23.0

### ISSUE THREAD

**Issue by vietj** [vert-x3/vertx-grpc#43]

**Comment by pmlopes** [comment]
@vietj grpc 1.23.0 uses netty 4.1.38 which will not align with the .39 plan... Should we force it to use .39 on our side? Or just let maven do it's alignment?

**Comment by vietj** [comment]
yes we should force to use it

**Comment by pmlopes** [comment]
```
[INFO] io.vertx:vertx-grpc:jar:3.8.1-SNAPSHOT
[INFO] +- io.vertx:vertx-core:jar:3.8.1-SNAPSHOT:compile
[INFO] |  +- io.netty:netty-common:jar:4.1.39.Final:compile
[INFO] |  +- io.netty:netty-buffer:jar:4.1.39.Final:compile
[INFO] |  +- io.netty:netty-transport:jar:4.1.39.Final:compile
[INFO] |  +- io.netty:netty-handler:jar:4.1.39.Final:compile
[INFO] |  |  \- io.netty:netty-codec:jar:4.1.39.Final:compile
[INFO] |  +- io.netty:netty-handler-proxy:jar:4.1.39.Final:compile
[INFO] |  |  \- io.netty:netty-codec-socks:jar:4.1.39.Final:compile
[INFO] |  +- io.netty:netty-codec-http:jar:4.1.39.Final:compile
[INFO] |  +- io.netty:netty-codec-http2:jar:4.1.39.Final:compile
[INFO] |  +- io.netty:netty-resolver:jar:4.1.39.Final:compile
[INFO] |  +- io.netty:netty-resolver-dns:jar:4.1.39.Final:compile
[INFO] |  |  \- io.netty:netty-codec-dns:jar:4.1.39.Final:compile
[INFO] |  +- com.fasterxml.jackson.core:jackson-core:jar:2.9.9:compile
[INFO] |  \- com.fasterxml.jackson.core:jackson-databind:jar:2.9.9.1:compile
[INFO] |     \- com.fasterxml.jackson.core:jackson-annotations:jar:2.9.9:compile
[INFO] +- io.vertx:vertx-codegen:jar:3.8.1-SNAPSHOT:compile
[INFO] |  \- org.mvel:mvel2:jar:2.3.1.Final:compile
[INFO] +- io.vertx:vertx-docgen:jar:0.9.2:compile
[INFO] +- io.grpc:grpc-netty:jar:1.23.0:compile
[INFO] |  \- io.grpc:grpc-core:jar:1.23.0:compile (version selected from constraint [1.23.0,1.23.0])
[INFO] |     +- com.google.code.gson:gson:jar:2.7:compile
[INFO] |     +- com.google.android:annotations:jar:4.1.1.4:compile
[INFO] |     +- io.perfmark:perfmark-api:jar:0.17.0:compile
[INFO] |     +- io.opencensus:opencensus-api:jar:0.21.0:compile
[INFO] |     \- io.opencensus:opencensus-contrib-grpc-metrics:jar:0.21.0:compile
[INFO] +- io.grpc:grpc-protobuf:jar:1.23.0:compile
[INFO] |  +- io.grpc:grpc-api:jar:1.23.0:compile
[INFO] |  |  +- io.grpc:grpc-context:jar:1.23.0:compile
[INFO] |  |  +- com.google.errorprone:error_prone_annotations:jar:2.3.2:compile
[INFO] |  |  +- com.google.code.findbugs:jsr305:jar:3.0.2:compile
[INFO] |  |  \- org.codehaus.mojo:animal-sniffer-annotations:jar:1.17:compile
[INFO] |  +- com.google.protobuf:protobuf-java:jar:3.9.0:compile
[INFO] |  +- com.google.api.grpc:proto-google-common-protos:jar:1.12.0:compile
[INFO] |  \- io.grpc:grpc-protobuf-lite:jar:1.23.0:compile
[INFO] +- io.grpc:grpc-stub:jar:1.23.0:compile
[INFO] +- com.google.guava:guava:jar:20.0:compile
[INFO] +- org.mortbay.jetty.alpn:jetty-alpn-agent:jar:2.0.9:test
[INFO] +- junit:junit:jar:4.12:test
[INFO] |  \- org.hamcrest:hamcrest-core:jar:1.3:test
[INFO] +- io.vertx:vertx-unit:jar:3.8.1-SNAPSHOT:test
[INFO] +- io.netty:netty-transport-native-epoll:jar:linux-x86_64:4.1.39.Final:test
[INFO] |  \- io.netty:netty-transport-native-unix-common:jar:4.1.39.Final:test
[INFO] \- io.netty:netty-transport-native-kqueue:jar:osx-x86_64:4.1.39.Final:test
```

**Comment by pmlopes** [comment]
Fixed by #44
